src/rpc/blockchain.cpp

```diff
@@ -2268,17 +2269,49 @@ class BlockFiltersScanReserver
     }
 };
 
+static bool CheckBlockFilterMatches(BlockManager& blockman, const CBlockIndex* pindex, const GCSFilter::ElementSet& needles)
```

Comment:
```suggestion
static bool CheckBlockFilterMatches(BlockManager& blockman, const CBlockIndex& block, const GCSFilter::ElementSet& needles)
```

Shouldn't this be `&`? (Passing in `nullptr` would be UB)

Reply:
Thanks, done.